Twig/Extension/PostMetaExtension.php

```diff
@@ -49,6 +49,7 @@ public function getFunctions()
     {
         return array(
             new \Twig_SimpleFunction('wp_get_post_meta', array($this, 'getPostMeta')),
+            new \Twig_SimpleFunction('wp_get_image_url_from_id', array($this, 'getImageUrlFromId')),
```

Comment:
Perhaps it should be simpler
```php
new \Twig_SimpleFunction('wp_get_image_url_from_id', 'wp_get_attachment_url')
```
didn't tested it because I'm too lazy to launch a VM and install all requirements, but I think it should work :)